etsin_finder/frontend/js/stores/view/packages.notifications.js

```diff
@@ -0,0 +1,87 @@
+import * as yup from 'yup'
+
+import { observable, action, makeObservable } from 'mobx'
+import axios from 'axios'
+import urls from '../../components/qvain/utils/urls'
+
+// TODO: Error messages
+const emailSchema = yup
```

Comment:
hyvä ajatus pistää yup schema tänne. Vois olla paikallaan että muutkin schemat siirtäis mobx:n yhteyteen. Löytyis helpommin.